test/parser.coffee

```diff
@@ -37,228 +37,253 @@ describe 'VASTParser', ->
             @response.should.be.an.instanceOf(VASTResponse)
 
         it 'should have merged top level error URLs', =>
-            @response.errorURLTemplates.should.eql ["http://example.com/wrapper-error", "http://example.com/error"]
+            @response.errorURLTemplates.should.eql ["http://example.com/wrapperA-error", "http://example.com/wrapperB-error", "http://example.com/error"]
 
         describe '#For the 1st ad', ->
+            ad1 = null
+
+            before (done) =>
+                ad1 = _response.ads[0]
```

Comment:
You should reset `ad1` to `null` in an `after()` hook to avoid potential side-effect on upcoming test suites. 
(+ I think the `done()` function should only be used when the hook includes asynchronous calls)